CapybaraLearning/spec/dashboard_button_spec.rb

```diff
@@ -0,0 +1,28 @@
+require_relative '../all_requires.rb'
+
+describe 'When testing demo.opensourcecms website' do
+  let(:login_page) { LoginPage.new }
+  let(:home_page) { HomePage.new }
+
+  before(:all) do
+    Capybara.configure do |config|
+      config.run_server = false
+      config.default_driver = :poltergeist
+    end
+  end
+
+  it 'Verify logging' do
+    login_page.load
+    home_page = login_page.log_in('opensourcecms', 'opensourcecms')
+    expect(home_page.loggined?).to be true
+  end
+
+  it 'Verify buttons are clickable' do
+    home_page.get_main_page_buttons.each do |button|
+      if button.visible?
+        home_page.find_button(button.text).click
```

Comment:
you should add some verification to make this a real verification, for example: expect(home_page.find_button(button.text).click).not_to raise_error

Reply:
ohh 'not_to raise_error' - I was looking for this method)